src/scss/_core.scss

```diff
@@ -484,6 +484,12 @@
   }
 }
 
+.swal2-label {
+  display: block;
+  margin: $swal2-label-margin;
+  text-align: $swal2-label-align;
```

Comment:
since it'll be flex element, change to `justify-content: $swal2-input-label-justify-content;`